staticd/static_zebra.h

```diff
@@ -28,9 +32,14 @@ extern void static_zebra_route_add(struct route_node *rn,
 				   struct static_path *pn, safi_t safi,
 				   bool install);
 extern void static_zebra_init(void);
+extern void static_zebra_stop(void);
```

Comment:
Put a comment on this indicating it's for test code otherwise the reaper is going to come for it at some point.

Reply:
done